### ISSUE TITLE
Open and save dialog opening behind main window

### ISSUE THREAD

**Issue by bradym** [microsoft/vscode#147400]
Issue Type: <b>Bug</b>

When clicking Ctrl-O to open a file or Ctrl-S or Ctrl-Shift-S to save a file, the dialog box to perform the action opens behind the editor window instead of on top. 

This started happening after updating to 1.66. I've verified that if I downgrade to version 1.65.2-1646927742 the issue is not present. After upgrading to version 1.66.0-1648620611 the issue is present, and it exists in 1.66.1-1649257842 and 1.66.2-1649664567 as well.

I'm on Ubuntu 20.04.4 and I install code using the microsoft apt repository, specifically `http://packages.microsoft.com/repos/code stable/main amd64 Packages`.

I uninstalled all of my extensions and removed everything from my settings.json file to verify that neither of those things could be the issue.

Here's a gif showing this behavior: ![vscodebug](https://user-images.githubusercontent.com/289182/163237062-35ef200d-bc62-4815-a022-4f00dda537b4.gif)

```
Version: 1.66.2
Commit: dfd34e8260c270da74b5c2d86d61aee4b6d56977
Date: 2022-04-11T07:49:24.808Z
Electron: 17.2.0
Chromium: 98.0.4758.109
Node.js: 16.13.0
V8: 9.8.177.11-electron.0
OS: Linux x64 5.16.11-76051611-generic
```

VS Code version: Code 1.66.2 (dfd34e8260c270da74b5c2d86d61aee4b6d56977, 2022-04-11T07:49:24.808Z)
OS version: Linux x64 5.16.11-76051611-generic
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz (12 x 2600)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>rasterization: disabled_software<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|1, 1, 1|
|Memory (System)|31.07GB (18.19GB free)|
|Process Argv|--unity-launch --crash-reporter-id fe9565f5-58c9-4868-9a15-44278a047c88|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|x11|
</details>Extensions: none<details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
pythonvspyl392:30443607
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
pythondataviewer:30285071
vscod805:30301674
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
pythonvsnew555:30457759
vscscmwlcmt:30465135
pynewfile477:30463512

```
</details>

**Comment by bradym** [comment]
Just confirmed this is not limited to the keyboard shortcuts, it also happens when choosing these options from the File menu:

![image](https://user-images.githubusercontent.com/289182/163245038-31a673bc-dc10-4f29-a9f3-3a4abc832c30.png)

**Comment by jherico** [comment]
I'm experiencing the same behavior since I switched back to using Linux as my main desktop in the past couple weeks.   Whenever I do something that will trigger a dialog I see a desktop toast that says something like 

```
Visual Studio Code

"Open File" is ready
```

and then I can get to the dialog by clicking on the toast or by using the switch-window shortcut key.  Currently running Ubuntu 22.04 pre-release, but I was seeing the same behavior in 20.04 before I upgraded yesterday.

![image](https://user-images.githubusercontent.com/1533642/163470229-cf3a908d-f4c0-429e-9eb7-bd33d54b3d31.png)
